index.html

```diff
@@ -54,5 +54,165 @@ <h6>LET'S CONNECT</h6>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+          <li>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </li>
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <button><h6>See Project</h6></button>
+          </li>
+        </ul>
+      </article>
+      <article class="card">
+        <a href="" class="snapshot">
+          <img src="/Worksection Pics/snap4.png" alt="" />
+        </a>
+        <ul class="leftblock2">
+          <li .primarytext>
+            <h1>Multi-Post Stories</h1>
+            <div class="frame">
+              <a href="" class="client"><h6 class="clienttext">Canopy</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="role"><h6 class="roletext">Back End Dev</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+          <li>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </li>
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <button><h6>See Project</h6></button>
+          </li>
+        </ul>
+      </article>
+    </section>
+    <section class="about">
+      <h1>
+        About <br />
+        Myself
+      </h1>
+
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+
+      <h6>LET'S CONNECT</h6>
+
+      <ul class="social-container">
+        <li>
+          <a href=""><img src="/Pictures/Social/github.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="/Pictures/Social/linkedin.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="/Pictures/Social/victory.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="/Pictures/Social/twitter.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="/Pictures/Social/m.png" alt="" /></a>
+        </li>
+      </ul>
+      <button class="btn1">Get my resume</button>
+    </section>
```

Comment:
- [ ]  According to the project requirements, the button should follow the button styling: Active, Hover, and Pressed. see the design below:
### Design
![figma](https://user-images.githubusercontent.com/81727988/192098154-0c471f87-a22b-4f52-9a87-5ea0a892e3b1.png)